Functions/xASL_stat_ComputeSpatialCoV.m

```diff
@@ -1,7 +1,7 @@
-function sCov = xASL_stat_ComputeSpatialCoV(imCBF,imMask,nMinSize,bPVC,imGM,imWM)
+function sCov = xASL_stat_ComputeSpatialCoV(imCBF, imMask, nMinSize, bPVC, imGM, imWM)
```

Comment:
Would be good to give this function also a `bParametric` option, we can do this later

Reply:
Done. Formally prepare and updated function calls.